[docs] mention reordering on charts (#40409)
metabase/metabase: docs/questions/sharing/visualizations/line-bar-and-area-charts.md

### Data

> Here's where you can choose the columns you want to plot on your x and y axes. This is mostly useful if your table or result set contains more than two columns, like if you're trying to graph fields from an unaggregated table. If your question has series breakouts, you'll be able to see them here too.

Reviewer: ```suggestion
In the **Data** tab, you can choose which columns you want to plot on your x and y axes, which is handy when your results have more than one column (like in a series breakout).
```

Does this capture it?

>  You can also add additional metrics to your chart by clicking the **Add another series** link below the y-axis dropdown, or break your current metric out by an additional dimension by clicking the **Add a series breakout** link below the x-axis dropdown (note that you can't add an additional series breakout if you have more than one metric/series).

Reviewer: ```suggestion
You can also:

- **Add another series** to add another time series to the chart.
- **Add a series breakout** to view your current metric by another dimension (though you can't add an additional series breakout if you have more than one metric/series).
```

> If you have multiple series or breakouts, you can reorder them by dragging the cards with the series or breakout name. On line charts and unstacked area charts, reordering will change the order of series in the legend. On bar charts and stacked area charts, reordering series or breakouts will also reorder the bars or areas.

Reviewer: ```suggestion
To reorder series and breakouts, drag and drop the sidebar cards to rearrange them. On line charts and unstacked area charts, reordering will change the order of series in the legend. On bar charts and stacked area charts, reordering series or breakouts will also reorder the bars or areas.
```
